core/consensus/beefy/impl/beefy_impl.cpp

```diff
@@ -393,9 +397,53 @@ namespace kagome::network {
     return outcome::success();
   }
 
+  outcome::result<void> BeefyImpl::fetchHeaders() {
+    std::unordered_map<primitives::BlockNumber, BlockDataToStore> blocksToStore;
```

Comment:
We don't need to put back blocks that are already in db,
we want `Synchronizer::fetchHeadersBack` to fetch and save them.
```suggestion
```

Reply:
fetchHeadersBack does not exist. Put it's signature at least, because I do not understand.